.circleci/workflows.yml

```diff
@@ -1220,6 +1220,12 @@ commands:
             else
               yarn binary-package --version $(node ./scripts/get-next-version.js)
             fi
+      - run:
+          name: Smoke Test the Cypress binary
+          command: |
+            source ./scripts/ensure-node.sh
+            node --version
+            yarn binary-smoke-test --version $(node ./scripts/get-next-version.js)
```

Comment:
Can you provide more context as to why this is broken out? Feels very risky to remove it from the build script. This could easily get missed/dropped over time. Also did you make corresponding changes in the trigger binary release workflow?

Reply:
We are splitting this out specifically for the publish binary changes that need to be made (https://github.com/cypress-io/cypress-publish-binary/pull/55). Unfortunately, with the upgrade to electron 27 and how they validate JSC files now, we have to make changes to the binary *after* it's been packaged and I wanted to make sure that the smoke tests are being run with the binary that is actually going out.